src/search.js

```diff
@@ -0,0 +1,50 @@
+import axios from"axios";
+
+const countryDisplayElement = document.getElementById("single-country");
+const inputData = document.getElementById("search-input");
+const searchForm = document.getElementById("country-search-form");
+
+function getInputData() {
+    const inputData = document.getElementById("search-input");
+    return inputData.value;
+}
+
+function getCurrency(result) {
+    let currencyInput = result.currencies;
+
+    if (currencyInput[0] && currencyInput[1]) {
+        return `${currencyInput[0].name} and ${currencyInput[1].name}`;
+    }
+    else if (currencyInput[0]) {
+        return currencyInput[0].name;
+    }
+}
+
+async function getCountryInfo() {
+    try {
+        const inputText = getInputData();
+        const countryResult = await axios.get("https://restcountries.com/v2/name/"+ inputText);
```

Comment:
Je kunt hier nog spreading toepassen, zodat je niet overal `countryResult.` vóór hoeft te zetten. Voorbeeldje:
```suggestion
        const { name, subregion, population, capital, currencies, languages } = await axios.get("https://restcountries.com/v2/name/"+ inputText);
```